lms/static/sass/views/_login-register.scss

```diff
@@ -356,6 +356,22 @@ $sm-btn-linkedin: #0077b5;
         text-transform: none;
         font-weight: 600;
         letter-spacing: normal;
+
+        .icon img {
+            width: auto;
+            height: auto;
+            max-height: 1.4em;
+            max-width: 1.4em;
+            margin-top: -4px;
+        }
```

Comment:
This selector duplicates the values set in _account.scss. If you use a class you could reuser the styles with the duplication.